src/sentry/issues/auto_source_code_config/code_mapping.py

```diff
@@ -86,7 +90,7 @@ def __init__(self, frame: Mapping[str, Any]) -> None:
         self.full_path = frame_file_path
         self.extension = get_extension(frame_file_path)
         if not self.extension:
-            raise UnsupportedFrameFilename("It needs an extension.")
+            raise NeedsExtension("It needs an extension.")
```

Comment:
This is an orthogonal change and it helps to differentiate from this code above:
<img width="717" alt="image" src="https://github.com/user-attachments/assets/0297e1cb-de93-424f-af87-0c9f271d1d65" />